packages/e2e/scripts/playwright/axe-clean.spec.ts

```diff
@@ -56,12 +56,59 @@ const testsToSkip = [
   'components-videoplayer-features--with-poster', // video makes this too flakey
   'components-videoplayer-features--without-branding', // video makes this too flakey
   'components-videoplayer--playground', // video makes this too flakey
-  'components-videoplayer--playground', // video makes this too flakey
   'recipes-feature-previews-level-1--level-one-side-by-side-enterprise', // video makes this too flakey
   'recipes-feature-previews-level-1--level-one-side-by-side', // custom, unrelated background image
   'components-eyebrowbanner-features--on-custom-background-dark', // custom, unrelated background image
   'components-eyebrowbanner-features--on-custom-background-light', // custom, unrelated background image
 ]
+
+const ignoreViolations = {
+  'landmark-one-main': {except: []}, // on most of the sotries we don't have a main landmark
```

Comment:
qq about this one @stamat... do we need this rule? So far most tests pass without needing to ignore it, so it feels like we should instead opt-in certain stories instead. wdyt?